### ISSUE TITLE
PECL still says latest release is version 0.6.0

### ISSUE THREAD

**Issue by sergeyklay** [jbboehr/php-psr#66]
```
$ pecl download psr-0.6.1

Failed to download pecl/psr, version "0.6.1", latest release is version 0.6.0, stability "stable",
use "channel://pecl.php.net/psr-0.6.0" to install
download failed
```

> https://pecl.php.net/package/psr/0.6.1

**Comment by jbboehr** [comment]
Yeah, kinda weird. Had a similar issue over here: https://github.com/jbboehr/php-mustache/issues/53

It seems like there can be a desync between the web interface and the REST api for the pecl command.

**Comment by jasny** [comment]
It's not in https://pecl.php.net/rest/r/psr/allreleases.xml. Maybe delete and re-publish (or publish new version).

**Comment by jbboehr** [comment]
Should be fixed with the 0.7.0 release, although 0.6.1 still fails to download, although it shows up in that endpoint...
